aio/content/marketing/contributors.json

```
@@ -904,5 +805,25 @@
     "website": "https://strongbrew.io",
     "bio": "Kwinten is a GDE for Angular with a passion for reactive coding. He works as a software engineer for Stackblitz. He shares his passion for Angular and coding through blogposts and training sessions with the company he co-founded called Strongbrew.",
     "groups": ["GDE"]
+  },
+  "juleskremer": {
+    "name": "Jules Kremer",
+    "picture": "juleskremer.jpg",
+    "twitter": "jules_kremer",
+    "bio": "Jules Kremer is the engineering manager for web frameworks at Google including Angular. Jules has over 20 years of experience in technology and has been a software engineer, product manager, developer evangelist, and competitive intelligence lead. Jules has a passion for supporting women in tech, yoga, hiking, cooking, and exploring the jungles of Maui.",
+    "groups": ["Angular"]
+  },
+  "agscott": {
```

We should use GitHub handles as keys (see [aio/content/marketing/README.md][1]), so this should be [atscott][2].

```suggestion
  "atscott": {
```

[1]: https://github.com/angular/angular/blob/4fa3dc2b2/aio/content/marketing/README.md#about-the-data
[2]: https://github.com/atscott